apps/client/src/app/components/account-detail-dialog/account-detail-dialog.component.ts

```diff
@@ -95,19 +96,9 @@ export class AccountDetailDialog implements OnDestroy, OnInit {
     this.dialogRef.close();
   }
 
-  public onAddAccountBalance({
-    balance,
-    date
-  }: {
-    balance: number;
-    date: Date;
-  }) {
+  public onAddAccountBalance(balance: CreateAccountBalanceDto) {
```

Comment:
Can you refactor to `accountBalance: CreateAccountBalanceDto`?

Reply:
Changed